### ISSUE TITLE
Can only open 2 panes when layout is main-horizontal

### ISSUE THREAD

**Issue by lingceng** [tmuxinator/tmuxinator#320]
It sometimes works with 3 panes opened, but most times it only open 2 panes, leaving the third `ls` pane disappear.

When I change the layout to even-horizontal. It always works fine. 3 panes all opened.

Anyone can help me find out why?

Here is my `~/.tmuxinator/store.yml`

    name: store
    root: ~/workspace/store

    tmux_options: -2

    windows:
      - editor:
          layout: main-horizontal
          panes:
            - vim Gemfile
            - bin/rails console
            - ls
      - server: rs

Other info:

`> uname -a`

    Linux zhongkx-ThinkPad-Edge-E440 3.16.0-41-generic #57~14.04.1-Ubuntu SMP Thu Jun 18 18:01:13 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

`> mux -v`

    tmuxinator 0.6.10

`>  tmux -V`

    tmux 2.0

I also use zsh with Oh-My-Zsh.

Here is my .tmux.conf

    ###########################
    #  Configuration
    ###########################

    # enable mouse scroll
    setw -g mode-mouse on

    # Following not make sense to me
    # Use tmux -2 instead.
    #
    # use 256 term for pretty colors
    set -g default-terminal "screen-256color"

    # enable vim get Ctrl and Alt command
    set-window-option -g xterm-keys on

    # increase scroll-back history
    set -g history-limit 5000

    # use vim key bindings
    setw -g mode-keys vi

    # decrease command delay (increases vim responsiveness)
    set -sg escape-time 1

    # increase repeat time for repeatable commands
    set -g repeat-time 500

    # start window index at 1
    set -g base-index 1

    # highlight window when it has new activity
    setw -g monitor-activity on
    set -g visual-activity on

    # re-number windows when one is closed
    set -g renumber-windows on

    ###########################
    #  Tmuxinator
    ###########################
    set-window-option -g pane-base-index 1

    ###########################
    #  Key Bindings
    ###########################

    # tmux prefix
    unbind C-b
    set -g prefix C-j

    # copy to system clipboard
    # bind -t vi-copy y copy-pipe 'xclip -in -selection clipboard'

    # reorder windows with shift left right
    bind-key -n S-Left swap-window -t -1
    bind-key -n S-Right swap-window -t +1

    # create 'v' alias for selecting text
    bind-key -t vi-copy v begin-selection

    # paste
    unbind C-p
    bind C-p paste-buffer

    # window splitting
    unbind %
    bind | split-window -h
    unbind '"'
    bind - split-window -v

    # resize panes
    bind -r H resize-pane -L 5
    bind -r J resize-pane -D 2
    bind -r K resize-pane -U 2
    bind -r L resize-pane -R 5

    # quickly switch panes
    unbind ^J
    bind ^J select-pane -t :.+

    # force a reload of the config file
    unbind r
    bind r source-file ~/.tmux.conf \; display "Reloaded!"

    unbind j
    bind -r j select-pane -D
    unbind k
    bind -r k select-pane -U
    unbind h
    bind -r h select-pane -L
    unbind l
    bind -r l select-pane -R

    ###########################
    # Status Bar
    ###########################
    # enable UTF-8 support in status bar
    set -g status-utf8 on

    # set refresh interval for status bar
    set -g status-interval 30

    # center the status bar
    set -g status-justify left

    # show session, window, pane in left status bar
    set -g status-left-length 40

    #set -g status-left '#[fg=red]Session:#S#[fg=red] Window:#I Pane:#P#[default]'
    set -g status-left ''

    # show hostname, date, time, and battery in right status bar
    #set-option -g status-right '#[fg=red]#H#[default] %I:%M'
    set-option -g status-right ''

    ###########################
    # Colors
    ###########################

    # color status bar
    set -g status-bg '#282829'
    set -g status-left '#[fg=green]#S#[default]'

    # highlight current window
    set-window-option -g window-status-current-fg black
    set-window-option -g window-status-current-bg white
    set-window-option -g window-status-format ' #I #W#F '
    set-window-option -g window-status-current-format ' #I #W#F '

    # set color of active pane
    set -g pane-border-fg default
    set -g pane-border-bg default
    set -g pane-active-border-fg green
    set -g pane-active-border-bg default

    # resize panes with mouse (drag borders)
    # set -g mouse-select-pane on
    set -g mouse-resize-pane on
    set -g mouse-select-pane on
    set -g mouse-select-window on

    # Supports `github_username/repo` or full git repo URLs
    set -g @tpm_plugins '              \
      tmux-plugins/tpm                 \
      tmux-plugins/tmux-sensible       \
      tmux-plugins/tmux-yank           \
      tmux-plugins/tmux-copycat        \
      tmux-plugins/tmux-resurrect      \
    '

    # Keep this line at the very bottom of tmux.conf.
    run-shell '~/.tmux/plugins/tpm/tpm'

**Comment by lingceng** [comment]
I think I found the reason. I run `mux debug store` and get the shell script.

```shell  
# Window "editor"
                                                              
tmux -2 send-keys -t store:1.1 '' C-m
tmux -2 send-keys -t store:1.1 vim\ Gemfile C-m
                                                              
tmux -2 splitw -c /home/zhongkx/workspace/store -t store:1
tmux -2 select-layout -t store:1 main-horizontal
                                                              
tmux -2 send-keys -t store:1.2 '' C-m
tmux -2 send-keys -t store:1.2 rc C-m
                                                              
tmux -2 splitw -c /home/zhongkx/workspace/store -t store:1
tmux -2 select-layout -t store:1 main-horizontal
                                                              
tmux -2 send-keys -t store:1.3 '' C-m
tmux -2 send-keys -t store:1.3 ls C-m
```

When I changed the second splitw to 
```shell
tmux -2 splitw -h -c /home/zhongkx/workspace/store -t store:1
````
And run the script after edited. It works. I get three panes in first window. 

I also tested some other case. When I change the `store.yml` with layout to main-vertical. It works.
When I change the debug script to only set layout for `store:1` at last. It works too.

It's obvious that we should use `splitw -h` for the third or more panes.

**Comment by lingceng** [comment]
This problem may caused by pull request:
Set tmux layout after each pane creation #131

**Comment by magicalbanana** [comment]
@lingceng Perhaps, can you provide a screen shot of your terminal when this happens?

Seems like an issue with splitting and readjusting panes as they are created.

**Comment by podung** [comment]
This should be fixed by #375.  Please re-open if it still persists after upgrading to 0.7.2.
